src/main/java/com/faforever/client/game/CustomGamesController.java

```diff
@@ -208,6 +226,10 @@ public void onTableButtonClicked() {
       Node root = gamesTableController.getRoot();
       populateContainer(root);
     });
+
+    if (!preferencesService.getPreferences().isShowGameDetailsSidePane()) {
+      onToggleSidePaneButtonClicked();
```

Comment:
I do not understand what happens here. Why causes a click on a table implicitly the click on a different button?
If `onToggleSidePaneButtonClicked` has multiple uses, then the method name is misleading.